pkg/crc/validation/validation.go

```diff
@@ -35,6 +36,11 @@ func ValidateDiskSize(value int) error {
 	if value < constants.DefaultDiskSize {
 		return fmt.Errorf("requires disk size in GiB >= %d", constants.DefaultDiskSize)
 	}
+	// https://github.com/code-ready/machine-driver-hyperkit/issues/18
+	if runtime.GOOS == "darwin" && value > constants.DefaultDiskSize {
+		return fmt.Errorf("Disk resizing is not supported yet on macOS")
```

Comment:
`Not yet` supported or `Not supported`

Yet might give an unrealistic idea about a timeline.